### PR TITLE
Move Expression creation to ExpressionUtils

### PR DESCRIPTION
**Pull request by timowest**
this PR is not backwards compatible, so no backport is provided

### CODE REVIEW

File: querydsl-core/src/main/java/com/querydsl/core/types/ExpressionUtils.java [hunk 1]
```diff
@@ -44,12 +47,197 @@ private UnderscoreTemplates() {
...
+    }
+
+    /**
+     * Create a new Operation expression
+     *
+     * @param operator
+     * @param args
+     * @return
+     */
+    public static <T> PredicateOperation predicate(Operator operator, ImmutableList<Expression<?>> args) {
```
Comment (Shredder121) [inline querydsl-core/src/main/java/com/querydsl/core/types/ExpressionUtils.java]: These `<T>`s don't occur anywhere in the method signature.

**Comment by timowest** [comment]
@johnktims @Shredder121 Is this good to merge?

**Comment by Shredder121** [comment]
It looks good to me.

**Comment by Shredder121** [comment]
I'll merge this then.

**Pull request merged**